src/builder.ts

```diff
@@ -1330,6 +1336,9 @@ export function buildTypes<
   schemaBuilder?: SchemaBuilder
 ): BuildTypes<TypeMapDefs> {
   const builder = schemaBuilder || new SchemaBuilder(config);
+  (config.plugins || [])
+    .filter((p) => p.onBuild)
+    .forEach((p) => () => p.onBuild!(types, builder));
```

Comment:
i’m not sure I see the value of being able to access the `types` at that stage, as it’s an incomplete representation of the total types, as opposed to adding some public helper methods like `.hasType(typeName)` to `builder`, and then calling `onBeforeBuild` in this line:

https://github.com/prisma-labs/nexus/blob/f37f20b20631931382ed12d61c5173a2e24e387d/src/builder.ts#L491-L495

Directly after `walkTypes`.